docs/sdk/capacity-credits.md

```diff
@@ -18,12 +18,12 @@ In order to increase your rate limit you'll need to mint an `Capacity Credits NF
 
 You’ll also need some 'testLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Habanero. `testLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).
 
-
 ```javascript
 const walletWithCapacityCredit = new Wallet("<your private key or mnemonic>");
 let contractClient = new LitContracts({
   signer: dAppOwnerWallet,
-  network: 'manzano'
+  network: 'manzano',
+  checkNodeAttestation: true
```

Comment:
checkNodeAttesation: true isn't needed, right?  because the SDK knows that it should turn it on for manzano + habanero unless the user manually turns it off?  or is that not in place on the SDK side?